src/index.html

```diff
@@ -5,9 +5,53 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+      >
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles/styles.css">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <main>
+      <div class="card" data-qa="card">
+        <img
+          class="card__image"
+          src="images/imac.jpeg"
+          alt="imac"
+        >
+
+        <div class="card__product">
+          <h1 class="card__product-name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+          <p class="card__product-code">Product code: 195434</p>
+        </div>
+
+        <div class="card__review">
+          <div class="stars stars--4">
```

Comment:
```suggestion
          <div class="card__stars stars stars--4">
```

Reply:
Why do we need class name 'card__stars' ?